class-wp-predis-decorator.php

```diff
@@ -55,6 +55,26 @@ public function transform_info( $info ) {
 	}
 
 	public function __call( $method_name, $args ) {
+		// Optionally prevent flushing on non-cli.
+		if ( in_array( strtolower( $method_name ), [ 'flushdb', 'flushall' ], true ) ) {
+			if ( function_exists( 'wp_debug_backtrace_summary' ) ) {
```

Comment:
Is there a case where this isn't defined?

Reply:
Mm I'm not sure, but I didn't want to cause fatals on such a low level function, however it _is_ included before object-cache is initted in `wp-settings.php` so could be safe to remove

Reply:
Revmoed this one